docker-compose.yml

```diff
@@ -101,9 +101,15 @@ services:
       traefik.http.routers.opencloud.rule: Host(`host.docker.internal`) && PathPrefix(`/`)
       traefik.http.routers.opencloud.entrypoints: opencloud
       traefik.http.services.opencloud.loadbalancer.server.port: 9200
-      traefik.http.routers.opencloud.middlewares: cors
+      traefik.http.routers.opencloud.middlewares: cors,redirect-caldav
+
+      traefik.http.middlewares.redirect-caldav.redirectregex.regex: '^/.well-known/caldav$'
+      traefik.http.middlewares.redirect-caldav.redirectregex.replacement: '/caldav/'
+      traefik.http.middlewares.redirect-caldav.redirectregex.permanent: true
```

Comment:
Since we're not accessing radicale via traefik, is this stuff really needed? Same with all the other traefik related lines below.